cli/src/trusted_base_cli/commands/pay_as_bid_proof.rs

```diff
@@ -70,11 +75,10 @@ pub(crate) fn pay_as_bid_proof(
 		Some(value) => {
 			let proof: MerkleProofWithCodec<_, _> =
 				MerkleProofWithCodec::decode(&mut &value[..]).unwrap();
-			Some(proof)
+			proof
```

Comment:
I'm wondering how the `proof` which is a type of `MerkleProofWithCodec` is converted to `String`?
Can you please say a few words about how this conversion is done? thanks

Reply:
I explicitly used `serde_json::to_string` above to ensure that we can use `serde_json::from_str` without an error.

Reply:
`serde_json::to_string`/`from_str` can only be used if you derive `Serialize`, `Deserialie`.

Reply:
I see you are converting in line 45 in the print statement. 
Then you use the output of the print statement in the `demo_energy_market.sh`.
The `proof` in line 78 is still the original decoded proof value.
is that correct?

Reply:
Yes, exactly. You can see in the function signature that we return a `MerkleProofWithCodec` struct.